R/utils_parametric_models.R

```diff
@@ -20,20 +29,28 @@ check_dist_params <- function(dist_params, distribution) {
     )
   }
 
-  if (!three_parametric && length(dist_params) != 2) {
+  if (!three_parametric && !one_parametric && length(dist_params) != 2) {
     stop(
       "A two-parametric distribution needs two parameters but",
       " 'dist_params' has length ", length(dist_params), ".",
       call. = FALSE
     )
   }
+
+  if (one_parametric && length(dist_params) != 1) {
+    stop(
+      "A one-parametric distribution needs one parameter but ",
+      sQuote(name_arg), " has length ", length(dist_params), ".",
```

Comment:
sQuote? Inconsistent with prior stop message

Reply:
Since this function is also used in other functions where the input argument is not named 'dist_params'  (e.g. `ml_estimation()` it is named 'start_dist_params') I've decided to throw a more informative message. 

`sQuote()` is now used in all stop messages of `check_dist_params()` (next commit)